spark/src/main/scala/org/apache/mahout/sparkbindings/drm/CheckpointedDrmSpark.scala

```diff
@@ -165,7 +168,15 @@ class CheckpointedDrmSpark[K: ClassTag](
       else if (classOf[Writable].isAssignableFrom(ktag.runtimeClass)) (x: K) => x.asInstanceOf[Writable]
       else throw new IllegalArgumentException("Do not know how to convert class tag %s to Writable.".format(ktag))
 
-    rdd.saveAsSequenceFile(path)
+    // rdd.saveAsSequenceFile(path)
+    // this is a (working) deprecated method used as a stop-gap while we investigate the shell issues
+    SparkContext.rddToSequenceFileRDDFunctions(rdd.asInstanceOf[RDD[(K, Vector)]]).saveAsSequenceFile(path)
+    // parameters to the non-depricated function are
+    // SequenceFileRDDFunctions(
+    //   self: RDD[(K, V)], // rdd.asInstanceOf[RDD[(K, Vector)] is ok here
+    //   _keyWritableClass: Class[_ <: Writable], // k2wFunc is ok here
+    //   _valueWritableClass: Class[_ <: Writable] // we need to have a factory for vector here, VectorWritable isn't
+    //                                             // acceptible as-is I think.
   }
```

Comment:
minimally dfsWrite should probably use a non-deprecated write function. This requires a factory for the vector used in drms, is this VectorWritable?